### ISSUE TITLE
Layer Properties x and y offsets not displaying correctly since last update

### ISSUE THREAD

**Issue by FawFuLJJ2** [Violet-CLM/MLLE#110]
![image](https://github.com/user-attachments/assets/f73c35a1-a921-479e-9d85-2fe1ddab56b5)

Tried using an x-offset 16 to place a torch inbetween tiles. MLLE is doing weird things displaying it incorrectly, sometimes it randomly fixes itself and sometimes it breaking again.

**Comment by Violet-CLM** [comment]
Does it depend on your zoom level? I suspect offsets are forgetting to account for zoom.

**Comment by FawFuLJJ2** [comment]
Yeah, nice find. Apparently the increased zoom levels (beyond what normally is capable in JCS) don't account for it.

**Comment by FawFuLJJ2** [comment]
A heads up that it is actually for any zoom level, both zoom in beyond 100% and zoom out. (I only mentioned zoom in before)
